phase2/kubeadm/do

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+set -x
+
+cd "${BASH_SOURCE%/*}"
+
+CONFIG_FILE="${CONFIG_JSON_FILE:-../../.config.json}"
+KUBEADM_INIT_PARAM_FILE=/etc/kubeadm/kubeadm_init_params.txt
+
+upgrade_master() {
+  KUBERNETES_VERSION=$(jq -r '.phase2.kubernetes_version' ${CONFIG_FILE})
+  UPGRADE_METHOD=$(jq -r '.phase2.kubeadm.master_upgrade.method' ${CONFIG_FILE})
+  MASTER=$(do_phase1 master-host)
+
+  update_host_kubeadm ${MASTER}
+  wait_nodes_ready
+
+  INIT_PARAM=$(execute_host ${MASTER} "sudo cat ${KUBEADM_INIT_PARAM_FILE}")
+  case "${UPGRADE_METHOD}" in
+    "init")
+      execute_host ${MASTER} "sudo kubeadm init --skip-preflight-checks --kubernetes-version ${KUBERNETES_VERSION} ${INIT_PARAM}"
+      ;;
+    "upgrade")
```

Comment:
`kubeadm-upgrade`
See: https://github.com/kubernetes/test-infra/pull/4495

Reply:
I think there has been a miscommunication. I am fine with the test using that in its own param. For the actual k8s anywhere config, the extra kubeadm is redundant. 

ie.
.phase2.kubeadm.master_upgrade.method=kubeadm-upgrade
.phase2.kubeadm.master_upgrade.method=init

Reply:
Well, as long as you've tested this end to end and what merged in https://github.com/kubernetes/test-infra/pull/4495 works, I'm ok with this